src/pages/myGeotab/addIns/mapAddIns.tsx

```diff
@@ -8,7 +11,7 @@ import mapAddinSizeOnMobileImage from "./../../../assets/images/mapAddins/map-ad
 import multipleMapAddinsOnTheMapPageImage from "./../../../assets/images/mapAddins/map-add-ins-docs-4.jpg";
 import { Link } from "react-router-dom";
 
-const Installation: ReactNode = <div className="paragraph">
+const installation: ReactNode = <div className="paragraph">
     <p>
         Map Add-ins are installed by uploading an <Link to="/myGeotab/addIns/developingAddIns">Add-in Configuration file</Link> into MyGeotab. Click on Administration -&gt; System... -&gt; System Settings -&gt; Add-Ins -&gt; New Add-In, and upload the Add-in Configuration file. The following is an example of a Map Add-in's Configuration file:
     </p>
```

Comment:
change CodeSample to self closing tag, also the code is too far indented, move it all over to the left